core/src/main/java/com/linecorp/armeria/server/docs/MethodInfo.java

```diff
@@ -197,6 +208,16 @@ public List<FieldInfo> parameters() {
         return parameters;
     }
 
+    /**
+     * Tells whether the {@link #parameters()} is used as the root when creating
+     * <a href="https://json-schema.org/">JSON schema</a>. The size of the {@link #parameters()} must be one
+     * if this returns {@code true}.
+     */
+    @JsonIgnore
+    public boolean useParameterAsRoot() {
```

Comment:
`AsRoot` tells how a serialized request message should be formatted. Overall, it looks good in the current context but I'd like to give it a more independent name from JSON serialization.
What do you think of renaming it to `requiresUnaryParameter()` or something similar that tells a single parameter should be used for the `MethodInfo`.

Reply:
> give it a more independent name from JSON serialization.

Isn't it all about JSON serialization? 😆 
I'm not sure where this will be useful and used out of this context. 😄 

>  renaming it to requiresUnaryParameter()

That's also a good name. 👍